account_invoice_fixed_discount/models/account_move_line.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2017 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
```

Comment:
I think it would be better to split the file into two. The first for account_move and the second for account_move_line